data/filedefs/filetypes.LESS.conf

```diff
@@ -0,0 +1,19 @@
+# For complete documentation of this file, please see Geany's main documentation
+[styling=CSS]
+[keywords=CSS]
+[indentation=CSS]
+
+[lexer_properties=CSS]
+lexer.css.less.language=1
+
+[settings=CSS]
+lexer_filetype=CSS
+tag_parser=CSS
+# default extension used when saving files
+extension=less
+
+# LESS has single-line comments in addition to CSS multi-line ones
+comment_single=//
+
+# MIME type
+mime_type=plain/text
```

Comment:
I'd suggest dropping this one if there's no specific mime type. If I remember correctly, the mime type is used for getting the right icon of the file type and better to get the CSS icon here.